kong.conf.default

```diff
@@ -83,14 +83,14 @@
 #proxy_listen_ssl = 0.0.0.0:8443 # Address and port on which Kong will accept
                                  # HTTPS requests if `ssl` is enabled.
 
-#admin_listen = 0.0.0.0:8001     # Address and port on which Kong will expose
+#admin_listen = 127.0.0.1:8001   # Address and port on which Kong will expose
                                  # an entrypoint to the Admin API.
                                  # This API lets you configure and manage Kong,
                                  # and should be kept private and secured.
 
-#admin_listen_ssl = 0.0.0.0:8444 # Address and port on which Kong will accept
-                                 # HTTPS requests to the admin API, if
-                                 # `admin_ssl` is enabled.
+#admin_listen_ssl = 127.0.0.1:8444  # Address and port on which Kong will accept
```

Comment:
Unfortunately, this update makes this line go over our 80 characters limit we try to respect for this file. For this particular case, it should be easy enough to simply move "accept" down a line (and ensure subsequent lines aren't over 80 char as well). Would you mind taking care of that? Thank you!

Reply:
No problems, Ill reformat that conf so its <80 wide.

Github q -- do i need to raise a new PR for a rebase against next?
Im fork I clone next branch from this repo, generate a diff from this PR, apply diff to forked branch, submit new PR?

Reply:
@pduldig-at-tw what should work is doing a `git rebase next` on this branch

Reply:
I added the upstream branch, then rebased it against kong/next, but it seems to have brought in all the other commits. Have I broken this PR?

Reply:
you might create a new branch and cherry pick your commits, see https://stackoverflow.com/questions/9853681/git-create-branch-from-range-of-previous-commits/9853814#9853814

Reply:
Thank you hisham! Most appreciated! I am still getting used to GH workflows here so thank you for your explanation too!